File: src/sdk/android/3/core-classes/memory-storage/georadiusbymember/snippets/georadiusbymember-1.java

kuzzle.memoryStorage.georadiusbymember("key", "Palermo", 200, "km", new ResponseListener<JSONObject[]>() {
  @Override
  public void onSuccess(JSONObject[] points) {
    // callback called once the action has completed
  }

  @Override
  public void onError(JSONObject error) {
  }
});

File: src/sdk/php/3/core-classes/memory-storage/georadiusbymember/snippets/georadiusbymember-1.php

<?php

use \Kuzzle\Kuzzle;


$kuzzle = new Kuzzle('localhost');

try {
  $points = $kuzzle->memoryStorage()->georadiusbymember('key', 'Palermo', 200, 'km');
}
catch (ErrorException $e) {

}
